addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -71,14 +75,14 @@
                                 </DropdownItem>
                             </t>
                         </Dropdown>
-                    </td>
-                </tr>
+                    </div>
+                </div>
                 <!-- Add / remove selection labels -->
-                <tr t-if="state.propertyDefinition.type === 'selection'">
-                    <td class="o_td_label align-top pe-2">
-                        <b>Values</b>
-                    </td>
-                    <td>
+                <div t-if="state.propertyDefinition.type === 'selection'" class="d-contents mb-3 mb-sm-0">
+                    <div class="text-900">
```

Comment:
Shouldn't the text-color be controlled by the `o_form_label` class?
If I'm wrong and you still need the `text-900` class I propose to apply it to the label directly